jina/main/parser.py

```diff
@@ -75,6 +75,8 @@ def set_hub_new_parser(parser=None):
                         help='where to output the generated project dir into.')
     parser.add_argument('--template', type=str, default='https://github.com/jina-ai/cookiecutter-jina-hub.git',
                         help='s directory containing a project template directory, or a URL to a git repository.')
+    parser.add_argument('--type', type=str, default='pod',
+                        help='(pod|app) create a template for executor hub pod or jina app using cookiecutter.')
```

Comment:
```suggestion
                        help='create a template for executor hub pod or jina app using cookiecutter.')
```
remove option list in help text as it is auto-generated and will be duplicated with `choices`

Reply:
done